### PR TITLE
installed gulp

### PR DESCRIPTION
**Pull request by dalla777**
Hey @kitt can I get your eyes on this and make sure there's nothing here that will break the build/cause issues? ty

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -18,3 +18,4 @@ exclude:
   - README.md
   - Vagrantfile
   - dev
+  - node_modules
```
Comment (kitt) [inline _config.yml]: Should add `package.json` and `gulpfile.js` here, too.  They shouldn't be served on the website.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "refresh-pdx",
+  "version": "1.0.0",
+  "description": "The home of RefreshPDX",
+  "main": "index.html",
```
Comment (kitt) [inline package.json]: Is this correct?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/refreshpdx/website.git"
+  },
+  "author": "SuperFriends",
+  "license": "ISC",
```
Comment (kitt) [inline package.json]: MIT? none?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "The home of RefreshPDX",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/refreshpdx/website.git"
+  },
+  "author": "SuperFriends",
```
Comment (kitt) [inline package.json]: <3

File: _config.yml (hunk 1 above)
Comment (kitt) [inline _config.yml]: Can you add how to use to the `README.md`?

**Comment by kitt** [comment]
Do we want this to run inside vagrant?

**Comment by dalla777** [comment]
Running in Vangrant, I have no idea. :| it's pretty easy to just npm install with all the correct versions, so I'm not sure if it's necessary

**Comment by kitt** [comment]
`sudo npm install -g gulp`  has this error:

```
 sudo npm install -g gulp 
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

**Comment by dalla777** [comment]
I always get npm warnings and I have no idea what to do about that, doesn't seem to have any effect on my workflow. also you are running a global gulp install... can you not just "npm install" from within the folder?

File: package.json (hunk 2 above)
Comment (dalla777) [inline package.json]: I have no idea. I changed entry to index.html from js.. we can say 2.0 I suppose for version

**Comment by kitt** [comment]
I'm looking to adding the gulp things to the vagrant install script so that no extra tasks are needed.  Grunt requires a global CLI version and a local version.  Gulp doesn't?

**Comment by dalla777** [comment]
I don't think that you do need a global version but I'm looking into

**Comment by kitt** [comment]
https://github.com/gulpjs/gulp/blob/master/docs/getting-started.md

`npm install --global gulp-cli`
`npm install --save-dev gulp`

**Comment by kitt** [comment]
The npm install errors are _shrug_ since we're running in vagrant and locally only.

**Comment by kitt** [comment]
👍🏼

**Comment by kitt** [comment]
People without gulp installed globally on their machines can use the `dev/start-gulp.sh` file, which requires a vagrant rebuild (`vagrant halt; vagrant destroy; vagrant up`)

**Pull request merged**